src/LibraryManager.Vsix/Json/SuggestedActions/UpdateSuggestedActionSet.cs

```diff
@@ -76,14 +77,16 @@ public override bool HasActionSets
         {
             var list = new List<ISuggestedAction>();
 
-            string latestStable = await catalog.GetLatestVersion(_provider.InstallationState.LibraryId, false, cancellationToken).ConfigureAwait(false);
+            string latestStable = LibraryNamingScheme.Instance.GetLibraryId(_provider.InstallationState.Name,
+                            await catalog.GetLatestVersion(_provider.InstallationState.LibraryId, false, cancellationToken).ConfigureAwait(false));
 
             if (!string.IsNullOrEmpty(latestStable) && latestStable != _provider.InstallationState.LibraryId)
```

Comment:
>latestStable [](start = 38, length = 12)

This will always be non-null and non-Empty now, because it will always contain the library name. I think it needs to check for null or empty on the version by itself. Probably the same for latestPre below.